### ISSUE TITLE
Warning: React does not recognize the `iconHovered` prop on a DOM element

### ISSUE THREAD

**Issue by goldylucks** [TeamWertarbyte/material-ui-rating#32]
See image below.

```bash
# cat client/node_modules/material-ui-rating/package.json | grep version      
"version": "4.1.0",
```

<details>
<summary>
Here's my Rating component, there's no `iconHovered` in there
</summary>

```jsx
import React, { Component } from 'react'
import PropTypes from 'prop-types'
import MuiRating from 'material-ui-rating'
import { withStyles } from '@material-ui/core'

class Rating extends Component {
  static propTypes = {
    classes: PropTypes.object.isRequired,
    value: PropTypes.number.isRequired,
    muiRatingClasses: PropTypes.object,
    readOnly: PropTypes.bool,
  }

  static defaultProps = {
    muiRatingClasses: {},
    readOnly: true,
  }

  render() {
    const {
      classes,
      value,
      muiRatingClasses,
      readOnly,
      ...restOfProps
    } = this.props
    return (
      <MuiRating
        classes={{
          root: classes.starsRoot,
          icon: classes.star,
          iconButton: classes.starButton,
          ...muiRatingClasses,
        }}
        value={value}
        readOnly
        {...restOfProps}
      />
    )
  }
}

const HEIGHT = 20

const styles = () => ({
  starsRoot: {
    display: 'flex',
    alignItems: 'center',
    height: HEIGHT,
  },
  starButton: {
    padding: 0,
    margin: 0,
    transform: 'scaleX(-1)',
    '& > span': {
      height: HEIGHT,
    },
  },
  star: {
    height: HEIGHT,
    width: 16,
  },
})

export default withStyles(styles)(Rating)

```

</details>

can you look into it? thanks!

![image](https://user-images.githubusercontent.com/6963212/59819839-3c403700-9332-11e9-90d1-9e8b7668e907.png)

**Comment by goldylucks** [comment]
same happens for `iconNormal` and `iconFilled`

**Comment by goldylucks** [comment]
Guys any news on this? this makes the package unusable :(

**Comment by leMaik** [comment]
@goldylucks Sorry, I somehow didn't see this issue. Found the bug, will fix it asap.

Edit: Still, that bug doesn't make the package "unusable". It just adds three warnings, which are annoying, but just warnings after all.

**Comment by leMaik** [comment]
Fixed version released: [v4.1.1](https://github.com/TeamWertarbyte/material-ui-rating/releases/tag/v4.1.1)

**Comment by goldylucks** [comment]
Thanks for the fix!

Made it unusable because the console log was full of warnings, we have a list of items with ratings ;)
